### ISSUE TITLE
XLights 16.1 lags 15-20 seconds on input on Mac M1 Max

### ISSUE THREAD

**Issue by Graham-s** [xLightsSequencer/xLights#4897]
**Describe the bug**
Having an issue with 15-20 second lag in XLights. To the point it is unusable.
If I don’t put anything in the sequence window, everything is responsive and fine. As soon as I put anything in the sequence, it causes the issue.
If I delete what I added so nothing is left, then it runs just fine again. Add something back, it slows right down.
Happy to try anything.
*** UPDATE: If I maximize the window it becomes MUCH more responsive so I can use it.

**To Reproduce**
Create a new sequence, drag any effect to any prop, issue occurs 100% of the time.
To consistently test, I create a new animation sequence, drag an effect to the sequence window, then click New Sequence. It should be almost instantaneous. Maximizing the window made that considerably better.

**Expected behavior**
Normal response time.

**Versions (please complete the following information):**
Running 2024.16.1
OS: Sequoia 15.0.1
HW: MBP, M1 Max, 32GB Ram

**Additional context**

This appears very similar to issue 3919 and 3827.

up to 18 second response time to anything once something is placed in the sequence window.
Have rebooted several times.
Turned on and off GPU rendering
Tried changing display to/from ProMotion>60Hz
Happens with a new sequence, old sequence, anything.

**Comment by derwin12** [comment]
Any difference in .17.1 ?

**Comment by Graham-s** [comment]
> Any difference in .17.1 ?

No, if anything it seems worse.It crashed the whole Mac when I went fullscreen.
I did discover that if I close the preview windows in a windowed environment, it is quite usable.

**Comment by cybercop23** [comment]
Is this still an issue? If so, please visit us in the Zoom room.
